roles/ssp/templates/config/config-1.18.php.j2

```diff
@@ -12,7 +12,11 @@ $config = [
      * where the a list of authentication sources (excluded admin) is displayed.
      * If no sources are found, you need to login as admin and check configuration.
      */
+{% if ssp_twig_template %}
+    'usenewui' => true,
+{% else %}
     'usenewui' => false,
+{% endif %}
```

Comment:
`'usenewui' => {{ ssp_twig_template }}`,

Reply:
Right! Very nice and elegant! Thanks!